.github/workflows/tests.yaml

```
@@ -40,6 +41,9 @@ jobs:
       - uses: "ramsey/composer-install@v3"
         with:
           dependency-versions: ${{ matrix.dependencies }}
+      # only until psalm 6 support is added for PHP 8.5
+      - run: composer remove --dev vimeo/psalm --no-update
+        if: matrix.php == '8.5'
```

I would remove this always; it's never needed in this job, the one for Psalm is a separate one. I would also remove CS-Fixer
```suggestion
      - run: composer remove --dev vimeo/psalm friendsofphp/php-cs-fixer --no-update
```